apps/src/pythonlab/PythonlabView.tsx

```diff
@@ -1,8 +1,16 @@
 // Pythonlab view
 import React from 'react';
+import PythonEditor from './PythonEditor';
+import moduleStyles from './pythonlab-view.module.scss';
+import SidePanel from './SidePanel';
 
 const PythonlabView: React.FunctionComponent = () => {
-  return <div>Welcome to Python lab!</div>;
+  return (
+    <div className={moduleStyles.pythonLab}>
```

Comment:
Super nit: If we standardize on `Pythonlab` and `pythonlab`, should we use that same convention here?

Reply:
yep we should, will update!